platform/core/src/services/MeasurementService/MeasurementService.ts

```diff
@@ -570,7 +573,10 @@ class MeasurementService extends PubSubService {
    *
    * @param {string} measurementUID The measurement uid
    */
-  remove(measurementUID: string): void {
+  remove(measurementUID: string | string[]): void {
```

Comment:
can you add a new method removeMany instead?

Reply:
I see you've also included that below for visibility. I think this could really confuse the API and potentially cause issues when the function name doesn't match the parameter. I'd prefer the panel, toolbar, or command to simply loop over the UIDs and run them, or perhaps use a concrete `removeMany` or `toggleVisibilityMany` definition for new methods.

Reply:
Definitely want to have a method to do it in the measurement service since it might send notifications in different ways based on how many things are getting changed and that needs to be known about in that method.